patterns/core/ProductDetailInformation/Buybox.js

```diff
@@ -45,11 +45,29 @@ export default function Buybox(props) {
   function handleAddToCart() {
     setLoading(true)
 
+    GTM.trackEvent({
+      event: 'add_to_cart',
+      ecommerce: {
+        items: variants.map(prepareTrackingItem),
```

Comment:
I think this is wrong. We are now tracking as if we would add all available variants to the cart when in reality we are only ever adding one of the products variant to the cart on the detal page.

Also the quantity is missing here